titus-api/src/main/java/com/netflix/titus/api/appscale/model/PolicyConfiguration.java

```diff
@@ -104,6 +106,7 @@ public Builder withTargetTrackingPolicy(TargetTrackingPolicy targetTrackingPolic
         }
 
         public PolicyConfiguration build() {
```

Comment:
Should we also check that both step and target aren't null (one, and only one) must be set too.

Reply:
Sounds good.